src/main/scala/com/microsoft/hyperspace/index/rules/RuleUtils.scala

```diff
@@ -113,6 +113,11 @@ object RuleUtils {
               IndexLogEntryTags.COMMON_SOURCE_SIZE_IN_BYTES,
               commonBytes)
 
+            entry.setTagValue(
```

Comment:
This function is for hybrid scan.

Reply:
sorry I don't understand why we can't use this function. Could you please explain why we need to limit its use to hybrid scan?

Reply:
For gold standard tests, there are no files present on disk so to test this functionality, I will need to create index files. Just trying to avoid that case

Reply:
This change is for "non" hybrid scan, but if hybrid scan is not enabled, this function is not called.

Reply:
oh i see what you are saying, from your previous comment i thought "setTagValue" was created specifically for hybrid scan.